apps/docs/content/components/Image.mdx

```diff
@@ -0,0 +1,89 @@
+
+```jsx live
+<Image
+  src="https://source.unsplash.com/random"
+  alt="Random Image"
+  as="picture"
+/>
+```
+
+### Aspect ratio
+
+The `aspectRatio` prop can be used to set the aspect ratio of the image. This is useful when the image is not the same aspect ratio as the container.
+
+```jsx live
+<Image
+  src="https://source.unsplash.com/random"
+  alt="Random Image"
+  aspectRatio="16:9"
+/>
+```
+
+### Height
+
+The `height` prop can be used to set the height of the image. This can be used along side the `aspectRatio` prop to create a responsive image the same size as other images.
+
+```jsx live
+<Image
+  src="https://source.unsplash.com/random"
+  alt="Random Image"
+  height={200}
+  aspectRatio="16:9"
+/>
+```
+
+### Width
+
+The `width` prop can be used to set the width of the image. This can be used along side the `aspectRatio` prop to create a responsive image the same size as other images.
+
+```jsx live
+<Image
+  src="https://source.unsplash.com/random"
+  alt="Random Image"
+  width={200}
+  aspectRatio="16:9"
+/>
+```
+
+## Component props
+
+### Image <Label>Required</Label>
+
+
+| name          | type                                    | default     | required | description                                                                                                                                                               |
+| ------------- | --------------------------------------- | ----------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `src`         | `string`                                |             | `true`   | Specifies the path to the image                                                                                                                                           |
+| `alt`         | `string`                                |             | `true`   | Specifies a text value explaining the nature of the image for users of Assistive Technolog                                                                                                                         y
+| `as`          | `img`, `picture`                        | `img`       | `false`  | Specification to create a picture component                                                                                                                               |
+| `aspectRatio` | `'1:1'`, `'16:9'`, `'16:10'`, `'4:3'`, `'custom'` | `undefined` | `false`  | Sets the image aspect ratio. A custom ratio can be provided in the design tokens.                                                                                         |
+
+| `height`      | `number`                                |             | `false`  | The height of the image element or its container if it has an aspect ratio                                                                                                |
```

Comment:
FYI The table seems to be breaking around here somewhere...

![Screenshot 2023-04-17 at 09 31 50](https://user-images.githubusercontent.com/13340707/232432077-71341f42-c844-414a-9934-89e75fc423b2.png)

Reply:
this should be resolved in this build

Reply:
I'm still seeing it unfortunately and can't see any upcoming new deployments.. when you say 'this build', has it been pushed up?

Reply:
"This build" means the current build pushed to GitHub for me. It seems by local prettier is adding a space so it keeps adding and removing. It should be fixed now in the newest build.